relay-server/src/actors/processor.rs

```diff
@@ -871,7 +871,10 @@ impl EnvelopeProcessorService {
     /// At the moment client reports are primarily used to transfer outcomes from
     /// client SDKs.  The outcomes are removed here and sent directly to the outcomes
     /// system.
-    fn process_client_reports(&self, state: &mut ProcessEnvelopeState) {
+    fn process_client_reports(
+        &self,
+        state: &mut ProcessEnvelopeState,
+    ) -> Result<(), ProcessingError> {
```

Comment:
This would fail the entire envelope if there's a single broken timestamp in one of the client reports. We should rather fail these individually, or normalize them (see my above comment).